test/integration/puppetlabs/services/master/environment_classes_int_test.clj

```diff
@@ -62,15 +62,18 @@
   ([env-name if-none-match]
    (let [opts (if if-none-match
                 {:headers {"If-None-Match" if-none-match}})]
-     (http-client/get
-      (str "https://localhost:8140/puppet/v3/"
-           "environment_classes?"
-           "environment="
-           env-name)
-      (merge
-       testutils/ssl-request-options
-       {:as :text}
-       opts)))))
+     (try
+       (http-client/get
+        (str "https://localhost:8140/puppet/v3/"
+             "environment_classes?"
+             "environment="
+             env-name)
+        (merge
+         testutils/ssl-request-options
+         {:as :text}
+         opts))
+       (catch Exception e
+         (throw (Exception. "environment_classes http get failed" e)))))))
```

Comment:
oh nm I understand, you just want to know for sure that the thing happened during the get.